src/pages/Assistance/Assistance.tsx

```diff
@@ -123,7 +123,9 @@ const Assistance = () => {
           : undefined,
         emailConfirm: !values.emailConfirm
           ? requiredError
-          : values.emailConfirm !== values.email
+          : values.emailConfirm !== values.email &&
+            values.email &&
+            values.emailConfirm.toLocaleLowerCase() !== values.email.toLocaleLowerCase()
```

Comment:
it's sufficient to test just the case fixed strings
```suggestion
          : values.email &&
            values.emailConfirm.toLocaleLowerCase() !== values.email.toLocaleLowerCase()
```